docs/dao-setup.md

```diff
@@ -113,6 +113,14 @@ Lite node mode:
 _Don't forget to use different rpcBlockNotificationPorts for different full node instances, otherwise only one node will receive the new block event forwarded to that port._
 
 
+In default setup you need to run seed node as **full node**:
+`./bisq-seednode --daoActivated=true --genesisBlockHeight=111 --genesisTxId=30af0050040befd8af25068cc697e418e09c2d8ebd8d411d2240591b9ec203cf --baseCurrencyNetwork=BTC_REGTEST --useDevPrivilegeKeys=true --useLocalhostForP2P=true --nodePort=2002 --appName=bisq-BTC_REGTEST_Seed_2002 --fullDaoNode=true --rpcUser=YOUR_USER_NAME --rpcPassword=YOUR_PW --rpcPort=18443 --rpcBlockNotificationPort=5120`
+
+Then both Alice and Bob and the arbitrator as **lite nodes**:
+* `./bisq-desktop --daoActivated=true --genesisBlockHeight=111 --genesisTxId=30af0050040befd8af25068cc697e418e09c2d8ebd8d411d2240591b9ec203cf --baseCurrencyNetwork=BTC_REGTEST --useDevPrivilegeKeys=true --useLocalhostForP2P=true --nodePort=4444 --appName=bisq-BTC_REGTEST_arbitrator`
```

Comment:
Your line about Alice and Bob able to be lite node is good.  Not necessary to repeat the command line parameters here, since it has already been specified in this section of the document.